german-council-of-economic-experts.csl

```diff
@@ -0,0 +1,310 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="de-DE" demote-non-dropping-particle="sort-only">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>German Council of Economic Experts</title>
+    <title-short>GCEE</title-short>
+    <id>http://www.zotero.org/styles/german-council-of-economic-experts</id>
+    <link href="http://www.zotero.org/styles/german-council-of-economic-experts" rel="self"/>
```

Comment:
Can you add a "documentation" link here to https://www.sachverstaendigenrat-wirtschaft.de/? See https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link